flink-table/flink-table-api-java-bridge/src/main/java/org/apache/flink/table/api/bridge/java/BatchTableEnvironment.java

```diff
@@ -442,8 +442,11 @@
 	 * </pre>
 	 *
 	 * @param connectorDescriptor connector descriptor describing the external system
+	 * @deprecated this method might be refactored or dropped in the next versions,
```

Comment:
It would be nice to answer the `why?` question for users. E.g. `The SQL {@code CREATE TABLE} DDL is richer than this part of the API. This method might be refactored in the next versions. Please use {@link #executeSql(String) executeSql(ddl)} to register a table instead.`